src/lib.rs

```diff
@@ -2,13 +2,13 @@
 
 extern crate proc_macro;
 
-use heck::SnakeCase;
+use heck::{CamelCase, KebabCase, MixedCase, ShoutySnakeCase, SnakeCase};
 use proc_macro::TokenStream;
 use quote::quote;
 use syn::*;
 use proc_macro2::{Ident, Span};
 
-#[proc_macro_derive(DbEnum, attributes(PgType, DieselType, db_rename))]
+#[proc_macro_derive(DbEnum, attributes(PgType, DieselType, DefaultValueStyle, db_rename))]
```

Comment:
I don't really like `DefaultValueStyle` as it doesn't suggest it has anything to do with the database. Something like `DbValueStyle` would be clearer

Reply:
Done!